builder/docker/tinygo/smoke.sh

```diff
@@ -44,7 +44,6 @@ if [ $? -ne 0 ]; then
 fi
 
 # Check the base image
-assert_file $base/Dockerfile.base $version 'branch \K.+(?= https://github.com/tinygo-org/tinygo.git)'
```

Comment:
Removed this line @jagger27 as it is impossible for both the base image and builder image to be updated at the same time.